src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
```

Comment:
The class name 'RightTriangle' suggests that it is a specific type of triangle. However, the class should not start with a verb. Consider renaming the class to something more indicative of its purpose without starting with a verb, such as 'TriangleWithRightAngle'.